.github/workflows/tox.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest] # All OSes pass except Windows because tests need Unix-only fcntl, grp, pwd, etc.
-        python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11", "pypy-3.8" ]
+        python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11", "3.12", "pypy-3.8", "pypy-3.10" ]
```

Comment:
Should we add pypy-3.9 here, too?

Reply:
https://www.pypy.org/download.html mentions pypy-3.9 and pypy-3.10, NOT pypy-3.8.

Reply:
@pajod can you make the change there?